src/styles.scss

```diff
@@ -8,3 +8,8 @@
 }
 
 @import '../projects/wvr-elements/src/lib/shared/styles/wvr-elements';
+
+html,
+body {
+  height: 100%;
```

Comment:
I do not think we should be imposing site wide styles from a component

Reply:
I do. Lets do it. But set it when a component is registered.

Reply:
What would break?

Reply:
I wonder, should this be moved to tamu-library-components CSS/SCSS?